can/interfaces/usb2can/serial_selector.py

```diff
@@ -16,21 +23,21 @@ def WMIDateStringToDate(dtmDate):
         strDateTime = strDateTime + dtmDate[7] + '/'
     else:
         strDateTime = strDateTime + dtmDate[6] + dtmDate[7] + '/'
-        strDateTime = strDateTime + dtmDate[0] + dtmDate[1] + dtmDate[2] + dtmDate[3] + " " + dtmDate[8] + dtmDate[
-            9] + ":" + dtmDate[10] + dtmDate[11] + ':' + dtmDate[12] + dtmDate[13]
+        strDateTime = strDateTime + dtmDate[0] + dtmDate[1] + dtmDate[2] + dtmDate[3] + ' ' + dtmDate[8] + dtmDate[
+            9] + ':' + dtmDate[10] + dtmDate[11] + ':' + dtmDate[12] + dtmDate[13]
     return strDateTime
 
 
 def serial():
-    strComputer = "."
+    strComputer = '.'
```

Comment:
Just because?

Reply:
I simply followed [this](https://stackoverflow.com/a/56190/3753684) "rule" because I personally like it and it was used in many other parts of this library as well (AFAIK).